localflavor/mx/mx_states.py

```diff
@@ -9,11 +9,11 @@
     ('BCN', _('Baja California')),
     ('BCS', _('Baja California Sur')),
     ('CAM', _('Campeche')),
+    ('CDMX', _('Ciudad de México')),
```

Comment:
Would be using the three letter airport code for Ciudad de México (MEX) a fix for this?